### DISCUSSION TITLE
Joining Builders Team

### DISCUSSION THREAD

**Discussion by git404x** [ROM-builders/temporary#14105] [Q&A]
Hi, can i join the team 

Github: https://github.com/iTZUDAY2312
Telegram: [@itz_me_unknown404 
](https://telegram.me/itz_me_unknown404)
 (Telegram acc has no name but acc is real)

Some Successful builds:
https://cirrus-ci.com/build/5211713418035200
https://cirrus-ci.com/build/5423616098566144
https://cirrus-ci.com/build/5059141222793216
https://cirrus-ci.com/build/4742908174663680

Sources: [miatoll]
[Device tree](https://github.com/iTZUDAY2312/device_xiaomi_miatoll) & [Device Common](https://github.com/iTZUDAY2312/device_xiaomi_sm6250-common)
[Vendor tree](https://github.com/iTZUDAY2312/vendor_xiaomi_miatoll) & [Vendor Common](https://github.com/iTZUDAY2312/vendor_xiaomi_sm6250-common)

i have cherry picked stuffs in device common tree and vendor common as well u can check [CrDroid](https://github.com/iTZUDAY2312/device_xiaomi_sm6250-common/commits/CrDroid) branch

Reply sir

**Comment by Apon77** [comment]
mismatched github and telegram name or username cant be added, because i cant mention you when needed!

**Reply by git404x** [reply]
If i use my name on telegram ? Then i can join right? 

My Telegram  [Itz_UDAY_404](https://telegram.me/itz_me_unknown404) 

Name is updated to my own
